tests/Behat/FeatureContext.php

```diff
@@ -187,6 +187,83 @@ protected function getLangcodeByName(string $language_name): string {
     throw new \Exception("Language name '$language_name' is not valid.");
   }
 
+  /**
+   * Visit a edit page by node title.
+   *
+   * @When I visit node :arg1 edit page
+   */
+  public function iVisitNodeEditPage($arg1) {
+    $node = $this->getNodeByTitle($arg1);
+    $this->visitPath("/node/{$node->id()}/edit");
+  }
+
+  /**
+   * Checks, that form field has specified value.
+   *
+   * @Then the :field field contains :value
+   */
+  public function theFieldContains($field, $value) {
+    $node = $this->getSession()->getPage()->findField($field);
```

Comment:
You need to first assert that the field exists, before trying to access its value. You can use assert session for this.